sormas-backend/src/test/java/de/symeda/sormas/backend/sample/SampleFacadeEjbTest.java

```diff
@@ -343,6 +344,111 @@ public void testSampleDeletion() {
 		assertTrue(getSampleTestFacade().getDeletedUuidsSince(since).contains(sampleTest.getUuid()));
 	}
 
+	@Test
+	public void testAllSamplesDeletion() {
+
+		Date since = new Date();
+
+		RDCFEntities rdcf = creator.createRDCFEntities("Region", "District", "Community", "Facility");
+		UserDto user = creator
+			.createUser(rdcf.region.getUuid(), rdcf.district.getUuid(), rdcf.facility.getUuid(), "Surv", "Sup", UserRole.SURVEILLANCE_SUPERVISOR);
+
+		//1st Case person
+		PersonDto firstCazePerson = creator.createPerson("FirstCase", "FirstPerson");
+		CaseDataDto firstCaze = creator.createCase(
+			user.toReference(),
+			firstCazePerson.toReference(),
+			Disease.EVD,
+			CaseClassification.PROBABLE,
+			InvestigationStatus.PENDING,
+			new Date(),
+			rdcf);
+		SampleDto firstSample =
+			creator.createSample(firstCaze.toReference(), new Date(), new Date(), user.toReference(), SampleMaterial.BLOOD, rdcf.facility);
+		SampleDto secondSample =
+			creator.createSample(firstCaze.toReference(), new Date(), new Date(), user.toReference(), SampleMaterial.BLOOD, rdcf.facility);
+		PathogenTestDto firstSamplePathogenTest = creator.createPathogenTest(
+			firstSample.toReference(),
+			PathogenTestType.MICROSCOPY,
+			firstCaze.getDisease(),
+			new Date(),
+			rdcf.facility,
+			user.toReference(),
+			PathogenTestResultType.POSITIVE,
+			"Positive",
+			true);
+		AdditionalTestDto firstSampleAdditionalTest = creator.createAdditionalTest(firstSample.toReference());
+
+		// Database should contain the created sample and sample test
+		assertNotNull(getSampleFacade().getSampleByUuid(firstSample.getUuid()));
+		assertNotNull(getSampleFacade().getSampleByUuid(secondSample.getUuid()));
+		assertNotNull(getSampleTestFacade().getByUuid(firstSamplePathogenTest.getUuid()));
+		assertNotNull(getAdditionalTestFacade().getByUuid(firstSampleAdditionalTest.getUuid()));
+
+		getSampleFacade().deleteAllSamples(Arrays.asList(firstSample.getUuid(), secondSample.getUuid()));
+
+		// Sample and pathogen test should be marked as deleted, additional test should be deleted
+		assertTrue(getSampleFacade().getDeletedUuidsSince(since).contains(firstSample.getUuid()));
+		assertTrue(getSampleFacade().getDeletedUuidsSince(since).contains(secondSample.getUuid()));
+		assertTrue(getSampleTestFacade().getDeletedUuidsSince(since).contains(firstSamplePathogenTest.getUuid()));
+		assertNull(getAdditionalTestFacade().getByUuid(firstSampleAdditionalTest.getUuid()));
+
+		//2nd Case person
+		PersonDto secondCazePerson = creator.createPerson("SecondCase", "SecondPerson");
```

Comment:
seems to be the same as the code above, why do you test the same thing twice?
if it's not the same then please create a new test method with proper naming